AppCenter/AppCenter/Internals/Util/MSEncrypter.m

```diff
@@ -100,17 +102,29 @@ - (NSString *_Nullable)decryptString:(NSString *)string {
   }
   NSString *result = nil;
```

Comment:
We probably want unit tests to decrypt nil or empty strings.

Reply:
I added a test to check for empty string. Nil string is not supposed to happen (see comment about nullability above), so no need to write a test for it.